concepts/numbers/introduction.md

```diff
@@ -0,0 +1,140 @@
+# Introduction
+
+## Numerical types
+
+[Numbers][numbers] can be integer, unsigned integer, or floating point types.
+Each comes in various "sizes", meaning how many bits it needs in memory.
+
+- Integers can be `Byte`, `Short`, `Int` or `Long`, respectively 8, 16, 32 and 64 bits.
+- Unsigned integers have a `U` prefix: `UByte`, `UShort`, `UInt` or `ULong`.
+- Floating point types are `Float` (32-bit) or `Double` (64-bit).
+
+Integer variables relying on type inference default to `Int`, even on 64-bit machines, but floating point variables default to `Double`.
+
+Other types can, of course, be specified, but there are a few syntactic shortcuts, and big integer literals become `Long` if they would overflow `Int`.
+
+```Kotlin
+val one = 1 // defaults to Int
+val threeBillion = 3_000_000_000 // Long, with optional underscores for clarity
+val oneLong = 1L // Long
+val oneByte: Byte = 1
+val oneDouble = 1.0 // defaults to Double
+val oneFloat = 1.0f //Float
+val lightSpeed = 3.0e8 // scientific notation (units of m/s)
+```
+
+Hexadecimal and binary literals are conventional: `0x7F` and `0b100101` respectively.
+Octal literals are not supported in Kotlin.
+
+## Arithmetic
+
+The basic arithmetic operators are the same as in many languages:
+
+```Kotlin
+4 + 3 // => 7
+4 - 3 // => 1
+4 * 3 // => 12
+4 / 3 // => 1  Int / Int always gives an Int
+-8 / 3 // => -2  Truncated towards zero
+-8.0 / 3 // => -2.6666666666666665
+```
+
+To get a floating point result from division, at least one of the numerator / denominator must be floating point.
```

Comment:
Isn't this true for any arithmetic operator in Kotlin? For example `3 + 2.0` will give `5.0` and `3 * 2.0` gives `6.0` (both results are doubles).